Empire/Empire.cpp

```diff
@@ -323,36 +321,35 @@ namespace {
             float group_drawdown = std::min(allocation, group_pp_available);
             allocated_pp[group] += group_drawdown;  // relies on default initial mapped value of 0.0f
             group_pp_available -= group_drawdown;
-            
-            float stockpile_drawdown = allocation == group_drawdown ? 
-                        0.0f : (allocation - group_drawdown);
-//XXX                        0.0f : (allocation - group_drawdown) / stockpile_conversion_rate;
-            DebugLogger() << "allocation " << group_pp_available;
-            DebugLogger() << "group_drawdown " << group_drawdown;
-            DebugLogger() << "stockpile_drawdown" << stockpile_drawdown;
+
+            float stockpile_drawdown = allocation == group_drawdown ?  0.0f : (allocation - group_drawdown);
+                                       // 0.0f : (allocation - group_drawdown) / stockpile_conversion_rate;
+            TraceLogger() << "allocation " << group_pp_available
+                          << " group_drawdown " << group_drawdown
+                          << " stockpile_drawdown" << stockpile_drawdown;
 
             // record allocation from stockpile
             // protect against any slight mismatch that might possible happen from multiplying
             // and dividing by a very very small stockpile_conversion_rate
             stockpile_drawdown = std::min(stockpile_drawdown, available_stockpile);
             if (stockpile_drawdown) {
-                DebugLogger() << "allocated " << stockpile_drawdown << " imperial PP";
+                TraceLogger() << "allocated " << stockpile_drawdown << " imperial PP";
                 allocated_stockpile_pp[group] += stockpile_drawdown;
                 available_stockpile -= stockpile_drawdown;
             }
-            
+
             //DebugLogger() << "... leaving " << group_pp_available << " PP available to group";
 
 
             // check for completion
             float block_cost = item_cost * queue_element.blocksize;
-            if (block_cost*(1.0f - queue_element.progress) - queue_element.allocated_pp < EPSILON)
-                queue_element.turns_left_to_next_item = 1;            
+            if (block_cost * (1.0f - queue_element.progress) - queue_element.allocated_pp < EPSILON)
```

Comment:
I wouldn't add spaces around `*`

Reply:
I've reverted this change, for future style reference I'm a little confused on the conditions where (and which) arithmetic operators should not be surrounded with spaces.